viewer/apiCrons.js

```diff
@@ -525,7 +525,7 @@ class CronAPIs {
         console.log('CRON', cq.name, cq.creator, '- start:', new Date(cq.lpValue * 1000), 'stop:', new Date(singleEndTime * 1000), 'end:', new Date(endTime * 1000), 'remaining runs:', ((endTime - singleEndTime) / (24 * 60 * 60.0)));
       }
 
-      Db.searchSessions(['sessions2-*', 'sessions3-*'], query, { scroll: internals.esScrollTimeout }, function getMoreUntilDone (err, result) {
+      Db.searchSessions(Db.defaultIndexPatterns(Config.getArray('queryExtraIndices', '')), query, { scroll: internals.esScrollTimeout }, function getMoreUntilDone (err, result) {
```

Comment:
Throughout the code the several places where `['sessions2-*', 'sessions3-*']` are passed into db.js have been replaced by `Db.defaultIndexPatterns` which takes an `extraIndices` array (pulled from the config's `queryExtraIndices` value) and merges its contents with the default `['sessions2-*', 'sessions3-*']` array. This merged array is returned.